src/main/java/AgeVerification.java

```diff
@@ -0,0 +1,15 @@
+public class AgeVerification {
+    public static void main(String[] args) {
+System.out.println(ageVerification(18));
+System.out.println(ageVerification(17));
+    }
+    public static String ageVerification (int age){
+        if (age >= 18) {
```

Comment:
wydaje mi się, że lepszym rozwiązaniem będzie po prostu zwrócenie "true/false".

Uprości Ci to testy (zamiast sprawdzać czy został zwrócony konkretny String będziesz sprawdzał Assertions.assertTrue()/asssertFalse())

ta metoda skróci się do 1 linijki :)


`return age >= 18`